common/common.go

```diff
@@ -102,6 +102,13 @@ func PodManifestPath(root string) string {
 	return filepath.Join(root, "pod")
 }
 
+// PodManifestLockPath returns the path in root to the Pod Manifest lock file.
+// This must be different from the PodManifestPath since mutations on the pod manifest file
```

Comment:
agreed, as a self-note, I should add more rationale in this section of the `pod-lifecycle.md` https://github.com/coreos/rkt/pull/3318/files#diff-7f4a1c7974386fb54394c2fd1f91e9d9R44 document